pull_request_template.md

```diff
@@ -1,12 +1,19 @@
-<!-- Description of the change, references like Slack or Trello links can provide more context -->
```

Comment:
Should we have a header, like:

```markdown
### Reference to Trello card
```

(I think we should try to refer Trello rather than Slack?)